lib/components/tab.vue

```diff
@@ -7,8 +7,7 @@
                    :aria-hidden="localActive ? 'false' : 'true'"
                    :aria-expanded="localActive ? 'true' : 'false'"
                    :aria-lablelledby="controlledBy || null"
-                   v-if="localActive || !lazy"
-                   v-show="localActive || lazy"
+                   v-show="localActive"
```

Comment:
Maybe if we move the lazy part to the tab content slot?  So the tab itself is always present 9but hiding with v-show), but the content inside can be taken out of DOM via a v-if?